frontend/pages/policies/ManagePoliciesPage/_styles.scss

```diff
@@ -180,4 +180,8 @@
       }
     }
   }
+
+  .empty-table__container {
+    max-width: none;
```

Comment:
Doesn't seem to make a difference one way or the other, but why include this?

Reply:
Fixed! I wrote this when there was a different Figma UI specced for team view which shouldn't be wrapped onto multiple lines, but that Figma piece was removed after questioning the design and I forgot to delete the related CSS.